code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -4599,105 +4599,83 @@
 	glass_name = "cup of xuizi pulque"
 	glass_desc = "A variation of Mictlanian pulque that is safe to consume for Unathi."
 
-//ZZZZOOOODDDDAAAAA
-
+// Zo'ra Sodas
 /decl/reagent/drink/zorasoda
-	name = "Zo'ra Soda Cherry"
-	description = "Zo'ra Soda, cherry edition. All good drinks come in cherry."
-	color = "#102000"
-	adj_sleepy = -2
-	caffeine = 0.4
-	taste_description = "electric cherry"
-	carbonated = TRUE
-
-/decl/reagent/drink/zorasoda/phoron
-	name = "Zo'ra Soda Phoron Passion"
-	description = "Reported to taste nothing like phoron, but everything like grapes."
-	color = "#863333"
-	adj_sleepy = -2
-	caffeine = 0.4
-	taste_description = "electric grape"
-	carbonated = TRUE
-
-/decl/reagent/drink/zorasoda/kois
-	name = "Zo'ra Soda K'ois Twist"
-	description = "Whoever approved this in marketing needs to be drawn and quartered."
-	color = "#dcd9cd"
-	adj_sleepy = -2
+	name = "Zo'ra Soda"
+	description = "Zo'ra Soda. You aren't supposed to see this."
+	color = "#000000"
+	adj_dizzy = -6
+	adj_drowsy = -4
+	adj_sleepy = -3
+	overdose = 45
 	caffeine = 0.4
-	taste_description = "sugary cabbage"
+	taste_description = "zo'ra soda"
 	carbonated = TRUE
 
-/decl/reagent/drink/zorasoda/kois/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
-	..()
+/decl/reagent/drink/zorasoda/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
 	if(alien != IS_DIONA)
-		M.make_jittery(5)
-
-/decl/reagent/drink/zorasoda/hozm
-	name = "Zo'ra Soda High Octane Zorane Might"
-	description = "It feels like someone is just driving a freezing cold spear through the bottom of your mouth."
-	color = "#365000"
-	adj_sleepy = -3
-	caffeine = 0.6
-	taste_description = "a full-body bite into an acidic lemon"
-	carbonated = TRUE
+		if(prob(5))
```

Comment:
prob(5) is probably too much and might cause some spamming. Did you test this?

Reply:
fixed.